### PR TITLE
docs: add release process document

### PR DESCRIPTION
**Pull request by phlogistonjohn**
Finally, I've gotten around to documenting the release process based on my notes from the release in June.

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Perform a final check that the image is built correctly. First, ensure that
+base images are up-to-date:
+
+```
+podman pull docker.io/golang:1.18
+podman pull registry.access.redhat.com/ubi8/ubi-minimal:latest
+```
+
+Note that the base images are expected to change over time. Make sure
+you pull the images that are used by the project's Dockerfile.
```
Comment (synarete) [inline docs/release-process.md]: Change: Dockerfile --> Containerfile

File: docs/release-process.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Apply a temporary pre-release tag to the image:
+```
+podman tag quay.io/samba.org/svcwatch:latest quay.io/samba.org/svcwatch:v0.3pre1
+```
+
+Log in to quay.io.
+
+Push the image using the temporary pre-release tag:
+```
```
Comment (synarete) [inline docs/release-process.md:53]: Add: 
```
podman login quay.io
...
```

**Review by synarete — APPROVED** [review]
LGTM.
See minor comments.

File: docs/release-process.md (hunk 1 above)
Comment (phlogistonjohn) [reply docs/release-process.md]: Fixed, thanks.

File: docs/release-process.md (hunk 2 above)
Comment (phlogistonjohn) [reply docs/release-process.md:53]: Added

File: docs/release-process.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Push the image using the temporary pre-release tag. Example:
+```
+podman push quay.io/samba.org/svcwatch:v0.3pre1
+```
+
+Wait for the security scan to complete. There shouldn't be any issues if you
+properly updated the base images before building. If there are issues and you
+are sure you used the newest base images, check the base images on quay.io and
+make sure the number of issues are are identical. The security scan can take
```
Comment (anoopcs9) [inline docs/release-process.md]:
```suggestion
make sure that number of issues are identical. The security scan can take
```

File: docs/release-process.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Draft a new set of release notes. Select the recently pushed tag. Start with
+the auto-generated release notes from GitHub (activate the `Generate release
+notes` button/link). Add an introductory section (see previous notes for an
+example). Add a "Highlights" section if there are any notable features or fixes
+in the release. The Highlights section can be skipped if the content of the
+release is unremarkable (e.g. few changes occurred since the previous release).
+
+Because this is a container based release we do not provide any build artifacts
+on GitHub (beyond the sources automatically provided there). Instead we we add
```
Comment (anoopcs9) [inline docs/release-process.md]:
```suggestion
on GitHub (beyond the sources automatically provided there). Instead we add
```

**Review by anoopcs9 — CHANGES_REQUESTED** [review]

File: docs/release-process.md (hunk 3 above)
Comment (phlogistonjohn) [reply docs/release-process.md]: The double are needs to be fixed. But I don't think that the other part of the fix is grammatical. I think "that the" would be OK though.

File: docs/release-process.md (hunk 4 above)
Comment (phlogistonjohn) [reply docs/release-process.md]: fixed

File: docs/release-process.md (hunk 3 above)
Comment (phlogistonjohn) [reply docs/release-process.md]: Updated

**Review by anoopcs9 — APPROVED** [review]

**Review by synarete — APPROVED** [review]
lgtm

**Review by spuiuk — APPROVED** [review]
ACK

**Comment by phlogistonjohn** [comment]
And merged months later in November. :snail:

**Pull request merged**